utils/deps.edn

```diff
@@ -1,9 +1,11 @@
-{:deps      {core                            {:local/root "../core"}
-             api                             {:local/root "../api"}
-             http-kit                        {:mvn/version "2.3.0"}
+{:deps      {core/core                       {:local/root "../core"}
```

Comment:
can we use `accelerated-text/core` here instead?

Reply:
also, noticed that in `api/deps.edn` we use `accelerated-text/accelerated-text    {:local/root "../core"}`
should choose one naming and use everywhere